packages/shared/src/components/fields/Textarea.tsx

```diff
@@ -103,7 +104,12 @@ function Textarea({
         id={inputId}
         ref={inputRef}
         onFocus={onFocus}
-        onBlur={onBlur}
+        onBlur={(e) => {
+          if (onExternalBlur) {
+            onExternalBlur(e);
+          }
+          onBlur();
+        }}
```

Comment:
This was a fix for focus or was missing all along on `Textarea`?

Reply:
Missing in textarea :)